From evaluation.py
cdl_dataset/dataset/Prepare_breakfast/g1.txt
Checking k1
Evaluator failed to find a plan
##############################
